mantle/kola/harness.go

```diff
@@ -180,6 +180,10 @@ var (
 			desc:  "systemd ordering cycle",
 			match: regexp.MustCompile("Ordering cycle found"),
 		},
+		{
+			desc:  "oom killer",
+			match: regexp.MustCompile("oom-killer"),
```

Comment:
Optional: `invoked oom-killer` would be slightly more specific.

Reply:
agree